### DISCUSSION TITLE
mise tasks comparison

### DISCUSSION THREAD

**Discussion by steveno** [jdx/mise#2215] [General]
First of all, I know that the `tasks` feature is still not finished, but has there been any look at the ecosystem when considering what to implement and how to do it? For example, I have historically been a user of [entr](https://github.com/eradman/entr). Would `mise tasks` be able to replace that? Could comparisons like this be documented some where?

**Comment by jdx** [comment]
you mean like `mise watch`?

**Reply by steveno** [reply]
Sorry. Yes. Right question, wrong tool.
